### PR TITLE
Add sash ECR repo in Bastion and allow across account access

### PR DESCRIPTION
**Pull request by scwatts**
Change overview:

* adds an ECR repo for sash in the Bastion account
* attaches existing policy for across account sharing/access

Also see:

* https://github.com/umccr/infrastructure/pull/299/
* https://github.com/umccr/infrastructure/pull/307

### CODE REVIEW

File: terraform/stacks/bastion_ecr/main.tf [hunk 1]
```diff
@@ -102,6 +102,24 @@ resource "aws_ecr_repository_policy" "star_align_nf_cross_accounts" {
...
+  image_scanning_configuration {
+    scan_on_push = false
+  }
+}
+
+resource "aws_ecr_repository_policy" "sash_cross_accounts" {
+  repository = aws_ecr_repository.sash.name
+  policy     = templatefile("policies/cross_accounts_policy_umccr.json", {})
+}
+
```
Comment (victorskl) [inline terraform/stacks/bastion_ecr/main.tf:122]: ah, looks like `star_align_nf_lifecycle` policy block got pushed down (i.e. as sash blocks get inserted)

Could you double check that please..?

Also, you would want `sash_lifecycle` policy for untagged images like others..?

**Review by victorskl — COMMENTED** [review]
Minor comment to attend, pls

File: terraform/stacks/bastion_ecr/main.tf (hunk 1 above)
Comment (scwatts) [reply terraform/stacks/bastion_ecr/main.tf:128]: Thanks for catching that, fixed!

**Review by victorskl — APPROVED** [review]
Cool.! LGTM. I will merge & apply to Bastion.

**Comment by victorskl** [comment]
Applied

**Comment by scwatts** [comment]
Great thank you! I'll be putting it to use in an hour or so 😁

**Pull request merged**